boom-app/src/views/PreView.vue

```diff
@@ -130,14 +130,16 @@ function postSingleObject(typeUrl: string, version: number, properties: MappedRe
   const dateNow = new Date(Date.now()).toISOString().split('T')?.at(0) ?? '2025-01-01'
 
   const body = {
-    type: convertToInternalDockerUrl(typeUrl).toString(),
+    type: typeUrl,
     record: {
       typeVersion: version,
       data: properties,
       startAt: dateNow,
     },
   }
-  return postRequest<ObjectCreateResponse>(body).then((res) => res.uuid)
+  return postRequest<ObjectCreateResponse>(body)
+    .then((res) => res.uuid)
+    .catch((error) => console.log(error))
```

Comment:
je vangt hier de errors af en logt ze, maar volgens komen mislukte entries nu niet meer in de lijst met errors op het scherm